.github/ISSUE_TEMPLATE.md/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: RFE+:pray:
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```

Comment:
What do you think of also asking if the proposer if they would be available to test the feature in their environment? Something like:

```
**Would you be able to assist in testing this functionality if implemented?**
```

Reply:
Makes sense, I'll add it to the template. Thanks! :)